src/behaviors/loading/loading.js

```diff
@@ -0,0 +1,28 @@
+(function() {
+'use strict';
+
+angular
+  .module('semantic.ui.behaviors.loading', [])
+
+  .directive('smLoading', smLoading);
+
+  smLoading.$inject = ['$parse'];
+
+  function smLoading($parse) {
+    return {
+      restrict:'A',
+      link: function(scope, element, attrs) {
+        scope.$watch(function() {
+          return $parse(attrs.smLoading)(scope);
+        }, function(loading) {
+          if (loading) {
+            element.addClass('loading');
```

Comment:
do would generalize behaviors, replacing the ngclass completely. Example using ngclass:
```html
<div ng-class="{'loading-big': ctrl.inProgress}"></div>
```